diff -urNr web-eid-orig/lib/libelectronic-id/lib/libpcsc-cpp/include/pcsc-cpp/pcsc-cpp.hpp web-eid/lib/libelectronic-id/lib/libpcsc-cpp/include/pcsc-cpp/pcsc-cpp.hpp
--- web-eid-orig/lib/libelectronic-id/lib/libpcsc-cpp/include/pcsc-cpp/pcsc-cpp.hpp	2024-09-05 12:18:55.000000000 +0200
+++ web-eid/lib/libelectronic-id/lib/libpcsc-cpp/include/pcsc-cpp/pcsc-cpp.hpp	2025-04-14 10:17:50.788869273 +0200
@@ -24,9 +24,11 @@
 
 #include "flag-set-cpp/flag_set.hpp"
 
+#include <cstdint>
+#include <limits>
 #include <memory>
 #include <vector>
-#include <limits>
+
 
 // The rule of five (C++ Core guidelines C.21).
 #define PCSC_CPP_DISABLE_COPY_MOVE(Class)                                                          \
