diff --git a/web-eid-app/lib/libelectronic-id/lib/libpcsc-cpp/include/pcsc-cpp/pcsc-cpp.hpp b/web-eid-app/lib/libelectronic-id/lib/libpcsc-cpp/include/pcsc-cpp/pcsc-cpp.hpp
index f6e2ac0..1234567 100644
--- a/web-eid-app/lib/libelectronic-id/lib/libpcsc-cpp/include/pcsc-cpp/pcsc-cpp.hpp
+++ b/web-eid-app/lib/libelectronic-id/lib/libpcsc-cpp/include/pcsc-cpp/pcsc-cpp.hpp
@@ -24,9 +24,11 @@
 
 #include "flag-set-cpp/flag_set.hpp"
 
+#include <cstdint>
+#include <limits>
 #include <memory>
 #include <vector>
-#include <limits>
+
 
 // The rule of five (C++ Core guidelines C.21).
 #define PCSC_CPP_DISABLE_COPY_MOVE(Class)                                                          \
